man/sbd.8.pod

```diff
@@ -333,6 +333,22 @@ prevent a successful crashdump from ever being written.
 
 Defaults to 240 seconds. Set to zero to disable.
 
+=item B<-r> I<N>
+
+Actions to be executed when sbd inquisitor loop detects a timeout or
+one of the watchers detects it's parent to have died.
+
+Set reboot-action to comma-separated combination of
```

Comment:
"reboot action" is fine if changing it is intrusive. "reset action" would also work.

BTW, do you think it would be worthwhile to have pacemaker's stonith-action override the reboot|off portion once it's known? Or maybe log a message if it's contradictory.

Reply:
Probably a good idea to take this into consideration for the auto-config stuff between sbd & pacemaker. Guess it is less critical as the timeout-issue as another node false assuming reboot instead of powerdown or vice-versa doesn't really hurt contrary to if it assumes a watchdog-timeout that is too short.

Reply:
What about timeout-action consistent over all occurrences?
This would as well set it better apart from actions triggered via message-slot.
Just not very descriptive for the case when it is used in case a watcher finds the inquisitor having gone away...

Reply:
timeout-action works for me